src/lib/helpers/config.ts

```diff
@@ -0,0 +1,68 @@
+import { z, ZodError } from "zod";
+import { parse } from "jsr:@std/toml";
+
+const ConfigSchema = z.object({
+    server: z.object({
+        port: z.number().default(8282),
+        host: z.string().default("127.0.0.1"),
+        secret_key: z.string().default("CHANGE_ME"),
```

Comment:
really cool :)! can't we use Deno.env.get in this? like `Deno.env.get("SERVER_SECRET_KEY")`

Reply:
Yep, that was something I was considering. I can do that.

Reply:
OK, I've pushed up a change that does this. Let me know if any should be different.